README.md

```diff
@@ -37,7 +37,7 @@ Maven Usage
 Use the following definition to use JLine in your maven project:
 
     <dependency>
-      <groupId>jline</groupId>
+      <groupId>org.jline</groupId>
       <artifactId>jline</artifactId>
```

Comment:
The groupId for 3.0.0.M1 is actually ```jline```.
The groupId ```org.jline``` is for 3.0.0.M2.
Could you please change the version at the same time ?